manage.py

```diff
@@ -2,14 +2,8 @@
 
 from __future__ import absolute_import
 from __future__ import unicode_literals
-import sys
```

Comment:
Can you add a comment here about why we don't have many imports?

Reply:
will do